descriptors/ModuleDescriptor-template.json

```
@@ -591,7 +591,7 @@
     },
     {
       "id": "users",
-      "version": "15.0"
+      "version": "15.0, 16.0"
```

I believe versions are space separated rather than comma separated. It might be that both are supported. It could be worth checking.
```suggestion
      "version": "15.0 16.0"
```